pom.xml

```diff
@@ -24,14 +24,16 @@
   </licenses>
   
   <properties>
-    <jenkins.version>1.566</jenkins.version>
-    <java.level>6</java.level>
+    <revision>1.5</revision>
+    <changelist>-SNAPSHOT</changelist>
+    <jenkins.version>1.625.3</jenkins.version>
```

Comment:
Not allowed with this parent POM version, [apparently](https://github.com/jenkinsci/plugin-pom/blob/master/CHANGELOG.md#340). See https://github.com/jenkinsci/plugin-pom/pull/167 by @daniel-beck. I wish the parent POM could somehow trigger a build failure if you got this wrong. Might require some creative use of the Enforcer plugin.

Anyway, I recommend just bumping up the baseline (and Java level).

Reply:
I don't see how my PR is related. It was simply to prevent SECURITY-595 related problems that developers would be unaware of in `hpi:run` and tests without `-Djenkins.version`.

Of course anything older than 2.1xy.z is probably unnecessary.

Reply:
@daniel-beck the changelog says

> Bump to Jenkins Core 2.138.4 minimum

Am I supposed to interpret that differently?